logic.js

```diff
@@ -0,0 +1,110 @@
+const Tweeter = function(){
+    const _posts = [
+        {
+            text: "First post!",
+            id: "p1",
+            comments: [
+                { id: "c1", text: "First comment on first post!" },
+                { id: "c2", text: "Second comment on first post!!" },
+                { id: "c3", text: "Third comment on first post!!!" }
+            ]
+        },
+        {
+            text: "Aw man, I wanted to be first",
+            id: "p2",
+            comments: [
+                { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+                { id: "c5", text: "Yeah, believe in yourself!" },
+                { id: "c6", text: "Haha second place what a joke." }
+            ]
+        }
+    ]
+
+    let postIdCounter = 2
+    let commentIdCounter = 6
+    let nextPostId = 3
```

Comment:
you do not need to have the `nextPostId` and the `nextCommentId` veriable. Just `++postIdCounter` to get the next id number